src/installer.ts

```diff
@@ -1,46 +1,12 @@
 import * as tc from '@actions/tool-cache';
+import * as core from '@actions/core';
 import * as path from 'path';
 import * as semver from 'semver';
 import * as httpm from '@actions/http-client';
 import * as sys from './system';
-import {debug} from '@actions/core';
+import os from 'os';
```

Comment:
Did you get the os to mock successfully?  In setup-node I had to do require('os') because that's the way the lib did it and it would only mock if it was loaded from the node loader in the same way.  If so, curious what the difference was.

Reply:
Yes, it mocks successfully.